etp-backend/src/main/clj/solita/etp/service/energiatodistus.clj

```diff
@@ -262,13 +263,37 @@
            " update conflicts with other concurrent update."))
     result))
 
+(defn find-required-properties [db versio]
+  (map (comp to-property-name :column_name)
+       (energiatodistus-db/select-required-columns db {:versio versio})))
```

Comment:
Olisiko hyvä yhtenäistää nimet ? Required properties kuulostaa hyvältä. Required columns kuulostaa kanta-constraintilta, ja tässä kyseessä on service-tason validointi ei niinkään kannan (vaikka validointisäännöt kannasta tuleekin).

Reply:
tuossa pitää huomata että tietokannassa pakolliset sarakkeet on tietokannan sarakkeiden nimillä ja ne konvertoidaan ulospäin property-nimiksi

Reply:
Aivan, totta tuo kyllä!